### PR TITLE
War card game

### PR DESCRIPTION
**Pull request by kerymov**

### CODE REVIEW

File: WarCardGame/app/src/main/java/com/example/warcardgame/Content.kt [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.shadow
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun Logo() {
```
Comment (yev-kanivets) [inline WarCardGame/app/src/main/java/com/example/warcardgame/Content.kt:17]: You could have these components in different files to better split them.

File: WarCardGame/app/src/main/java/com/example/warcardgame/GameScreen.kt [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+                }
+            )
+        }
+    }
+}
+
+private fun generateRandomCard(cardItems: List<Card>) =
+    cardItems[Random.nextInt(cardItems.lastIndex + 1)]
+
+private fun check(firstCard: Card, secondCard: Card): Winner {
```
Comment (yev-kanivets) [inline WarCardGame/app/src/main/java/com/example/warcardgame/GameScreen.kt:95]: You could simplify to `private fun check(firstCard: Card, secondCard: Card) = when {`

File: WarCardGame/app/src/main/java/com/example/warcardgame/ui/theme/Theme.kt [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    primaryVariant = Purple700,
+    secondary = Teal200,
+)
+
+private val LightColorPalette = lightColors(
+    primary = Purple500,
+    primaryVariant = Purple700,
+    secondary = Teal200,
+
+    /* Other default colors to override
```
Comment (yev-kanivets) [inline WarCardGame/app/src/main/java/com/example/warcardgame/ui/theme/Theme.kt:20]: Try to not leave comments in the production code.

**Review by yev-kanivets — APPROVED** [review]
Great job 👍 Not much to add 😉

**Comment by kerymov** [comment]
Ok, thanks)

**Pull request merged**